okapi/services/caches/search/all/docs.xml

```diff
@@ -262,6 +262,21 @@
         IDs. You should not use them. This parameter was implemented for
         internal use.</p>
     </opt>
+    <opt name='awarded_status' default='either'>
+        <p>Includes information about a "cache of the week/month etc." award
```

Comment:
"Includes" suggests that using this parameter includes something into the response. It's actually the opposite. I'd suggest some rewording, e.g. "Allows to filter caches based on awards it has been granted."